### DISCUSSION TITLE
[Discord] Mee6 warning DMs feel unnecessary

### DISCUSSION THREAD

**Discussion by UnePomme** [chromiecraft/chromiecraft#7062] [Suggestions]
We already have the warning message in chat. The DMs from Mee6's warnings are quite annoying as the warning filter is quite sensitive.
![image](https://github.com/chromiecraft/chromiecraft/assets/44067875/1cee7d49-606b-431c-83b6-0b3c1fa83609)

**Comment by ChaoticMe** [comment]
I fully agree. Can MEE6 be toned down a little?
